docs/usage.rst

```diff
@@ -606,8 +618,10 @@ A list of Python modules that are known to not work correctly with
 - the ``Pillow`` image library does not work with pyfakefs at least if writing
   JPEG files (see `this issue <https://github.com/jmcgeheeiv/pyfakefs/issues/529>`__)
 - ``pandas`` (the Python data analysis library) uses its own internal file
-  system access, written in C, and does therefore not work with pyfakefs
-  (see `this issue <https://github.com/jmcgeheeiv/pyfakefs/issues/528>`__)
+  system access, written in C, and does therefore not work with pyfakefs out
```

Comment:
system access written in C. Thus much of ``pandas`` will not work with ``pyfakefs``.
   Having said that, ``pyfakefs`` patches ``pandas`` so that many of the ``read_xxx``
   functions, including ``read_csv`` and ``read_excel``, do work with the fake
   file system. If you use only these functions, ``pyfakefs`` will work with ``pandas``.